packages/perseus/src/widgets/radio/base-radio.jsx

```diff
@@ -363,60 +379,7 @@ class BaseRadio extends React.Component<$FlowFixMe, $FlowFixMe> {
         return this.props.apiOptions.isMobile || this.props.deselectEnabled;
     };
 
-    /**
-     * Find the choice at the given index (wrapped, if necessary), focus it,
-     * uncheck the old choice, and check the new choice (unless it's crossed
-     * out, in which case we only uncheck the old choice).
-     *
-     * This is very similar to standard up/down arrow behavior, except that we
-     * don't select crossed-out choices.
-     *
-     * Handles `goToPrevChoice` and `goToNextChoice` calls from our children.
-     */
-    goToChoice: (number) => void = (newChoiceIndex) => {
```

Comment:
A lot of the `goToXYZ` stuff seemed unused. I pulled it out of Choice and the removal rippled out a bit. I feel like I did my due diligence before removing it and the tests still passed. 🤷

Reply:
I don't think we can actually rely on the tests passing as an indicator of whether or not it's being used.  If `goToChoice` is not being called by anything else then that's a good indicator.  The problem with relying solely on the tests to determine dead code is just b/c it isn't covered doesn't mean that it isn't used in production.

Reply:
I had a look at the code and it looks like `goToChoice` is indeed not being used.